tools/quake3/q3map2/map.c

```diff
@@ -1086,7 +1086,7 @@ static void ParseRawBrush( qboolean onlyLights ){
 		side->planenum = planenum;
```

Comment:
@TTimo: I'm not sure if we want to store `-1` here. The alternative would be to leave `side->planenum` uninitialized, which is also bad.